### DISCUSSION TITLE
Sonoff POW - Do not switch relay during update

### DISCUSSION THREAD

**Discussion by DefCon-CC** [arendst/Tasmota#10187] [Q&A] [answered]
Hello,

is it possible that the relay is switched on all the even during update of the Tasmota Firmware?

Greets

Patrick

**Comment by ascillato** [comment, marked as answer]
Hi, no. During the update process the relays are not switched.

**Comment by Jason2866** [comment]
If you upgrade from a very old version it can happen that the relay blinks.

**Comment by DefCon-CC** [comment]
Ok, because I'm running a Server behind the Sonoff POW which I do not want to be restarted. I suppose that after the update of the Tasmota FW the device will perform a reset and will go through init routine. I think there is no way to bypass this?!

**Comment by sfromis** [comment]
I've done many upgrades of the similar Sonoff Pow R2 without any relay clicking, no interruptions to the powered computer. Many versions ago, the relay would click during restarts, leading to a high risk of outages when upgrading.
